cypress/e2e/accessibility.cy.ts

```diff
@@ -1,41 +1,39 @@
-import driver from "./driver";
+import MaputnikDriver from "./driver";
 
 describe("accessibility", () => {
-    // skipped due to the following issue with cypress: https://github.com/cypress-io/cypress/issues/299
-  describe.skip("skip links", () => {
+  let { beforeAndAfter, given, when, get, should } = new MaputnikDriver();
+  beforeAndAfter();
+
+  describe("skip links", () => {
     beforeEach(() => {
-        driver.beforeEach();
-        driver.setStyle("layer");
+      when.setStyle("layer");
     });
-  
+
     it("skip link to layer list", () => {
-      const selector = driver.getDataAttribute("root:skip:layer-list");
-      driver.isExists(selector);
-      driver.typeKeys('{tab}');
-      driver.isFocused(selector);
-      driver.click(selector);
-  
-      driver.isFocused("#skip-target-layer-list");
+      const selector = "root:skip:layer-list";
+      should.isExists(selector);
+      when.tab();
+      should.beFocused(selector);
+      when.click(selector);
+      should.beFocused("skip-target-layer-list");
     });
-  
+
     it("skip link to layer editor", () => {
-      const selector = driver.getDataAttribute("root:skip:layer-editor");
-      driver.isExists(selector);
-      driver.typeKeys('{tab}{tab}');
-      driver.isFocused(selector);
-      driver.click(selector);
-  
-      driver.isFocused("#skip-target-layer-editor");
+      const selector = "root:skip:layer-editor";
+      should.isExists(selector);
+      when.tab().tab();
+      should.beFocused(selector);
+      when.click(selector);
+      should.beFocused("skip-target-layer-editor");
     });
-  
+
     it("skip link to map view", () => {
-      const selector = driver.getDataAttribute("root:skip:map-view");
-      driver.isExists(selector);
-      driver.typeKeys('{tab}{tab}{tab}');
-      driver.isFocused(selector);
-      driver.click(selector);
-  
-      driver.isFocused(".maplibregl-canvas");
+      const selector = "root:skip:map-view";
+      should.isExists(selector);
+      when.tab().tab().tab();
+      should.beFocused(selector);
+      when.click(selector);
+      should.canvasBeFocused();
```

Comment:
nit picking: Why does canvas get "special treatment" and the others don't? i.e. why doe the other tests has `should.beFocused("skip-target-layer-editor")` and not `should.layerEditorBeFocused`?